src/clientsettingsdlg.cpp

```diff
@@ -35,6 +35,16 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
 {
     setupUi ( this );
 
+#if defined ( Q_OS_IOS )
+    // IOS needs menu to close
+    QMenuBar* pMenu     = new QMenuBar ( this );
+    QAction *action = pMenu->addAction (tr ( "&Close" ));
```

Comment:
```suggestion
    QAction *action = pMenu->addAction ( tr (  "&Close"  ) );
```

Not exactly sure, but I think that's the correct style here.